zenodo/modules/spam/config.py

```diff
@@ -93,5 +93,8 @@ def default_spam_handling(deposit=None, community=None):
 # Preload spam model on Celery app initialization
 ZENODO_SPAM_MODEL_PRELOAD = False
 
-ZENODO_SPAM_DOMAINS_FILEPATH = None
+ZENODO_SPAM_DOMAINS_FORBIDDEN_LIST_FILEPATH = None
 """Path to a file with the list of forbidden email domains."""
+
+ZENODO_SPAM_DOMAINS_SAFELIST_FILEPATH = None
+"""Path to a file with the list of safelisted domains."""
```

Comment:
```suggestion
ZENODO_SPAM_DOMAINS_FORBIDDEN_PATH = None
"""Path to a file with the list of forbidden email domains."""

ZENODO_SPAM_DOMAINS_SAFELIST_PATH = None
"""Path to a file with the list of safelisted domains."""
```

Since we're renaming, let's go for something shorter too :)